driver-core/src/test/java/com/datastax/driver/core/TestListener.java

```diff
@@ -28,7 +29,13 @@
     private int test_index = 0;
 
     static {
-        System.out.println("[CCMBridge] Using Cassandra version: " + CCMBridge.CASSANDRA_VERSION + " (install arguments: " + CCMBridge.CASSANDRA_INSTALL_ARGS + ")");
+        if (CCMBridge.isDSE()) {
+            System.out.println("[CCMBridge] Using DSE version: " + CCMBridge.getDSEVersion() + " (C* " +
+                    CCMBridge.getCassandraVersion() + ", install arguments: " + CCMBridge.getInstallArguments() + ")");
+        } else {
+            System.out.println("[CCMBridge] Using Cassandra version: " + CCMBridge.getCassandraVersion() +
```

Comment:
Updated and changed this to use a logger instead of println.  I also moved this block to CCMBridge as that seems like a better home for it.  It will also defer parsing to when CCMBridge is first referenced instead of in TestListener (which may have some negative impacts, will observe test results).  output now looks like:

**DSE**
```
    0      [main] INFO  com.datastax.driver.core.CCMBridge  - Tests requiring CCM will use DSE version 4.8.3 (C* 2.1.11, install arguments: --install-dir=/Users/atolbert/Documents/Projects/dse-4.8.3 --dse)
```

**Cassandra**
```
    0      [main] INFO  com.datastax.driver.core.CCMBridge  - Tests requiring CCM will use Cassandra version 2.1.12 (install arguments: -v2.1.12)
```